backend/src/Designer/Filters/IO/IoExceptionFilterAttribute.cs

```diff
@@ -28,5 +29,9 @@ public override void OnException(ExceptionContext context)
             // TODO: Implement custom IO exceptions Error Codes
             context.Result = new ObjectResult(ProblemDetailsUtils.GenerateProblemDetails(context.Exception, IoErrorCodes.ResourceNotFound, HttpStatusCode.NotFound)) { StatusCode = (int)HttpStatusCode.NotFound };
         }
+        if (context.Exception is InvalidExtensionImageUploadException)
+        {
+            context.Result = new ObjectResult(ProblemDetailsUtils.GenerateProblemDetails(context.Exception, IoErrorCodes.UploadedImageNotValid, HttpStatusCode.BadRequest)) { StatusCode = (int)HttpStatusCode.BadRequest };
+        }
     }
```

Comment:
I wonder if this should not rather be placed in `AppDevelopmentExceptionFilterAttribute` 🤔